OmiseSwiftTests/ChargesOperationFixtureTests.swift

```diff
@@ -402,6 +402,65 @@ class ChargesOperationFixtureTests: FixtureTestCase {
         waitForExpectations(timeout: 15.0, handler: nil)
     }
     
+    func testFPXChargeCreate() {
+        let expectation = self.expectation(description: "FPX Charge create")
+        let fpxParams = FPX(bank: FPX.FPXBank.affin, email: "test@omise.co")
+        let createParams = ChargeParams(value: Value(amount: 2500, currency: .myr),
+                                        sourceType: .fpx(fpxParams))
+        
+        _ = Charge.create(using: testClient, params: createParams) { (result) in
+            defer { expectation.fulfill() }
+            
+            switch result {
+            case let .success(charge):
+                XCTAssertNotNil(charge)
+                XCTAssertEqual(charge.value.amount, 2500)
+                XCTAssertEqual(charge.source?.paymentInformation, .fpx(fpxParams))
+                switch charge.source?.paymentInformation {
```

Comment:
This is duplicated. You can remove this `switch` because you test the same with the line above.